presto-main/src/main/java/io/prestosql/sql/planner/Partitioning.java

```diff
@@ -89,12 +89,25 @@ public PartitioningHandle getHandle()
 
     public Set<Symbol> getColumns()
```

Comment:
consider renaming it to `getPartitioningColumns` to make it more explicit about what it represents